tests/detect-ldap-result/test.rules

```diff
@@ -1 +1,2 @@
 alert ldap any any -> any any (msg:"Test LDAP result code"; ldap.responses.result_code:size_limit_exceeded; sid:1;)
+alert ldap any any -> any any (msg:"Test LDAP error message"; ldap.responses.error_message; content:"Size limit exceeded"; sid:2;)
```

Comment:
So, both of them are the same things, one a sticky buffer and one a value?
Was expecting `result_code` to be an int only.. which it does seem to accept as well. Is there a special name for the keywords that accept a string (?) value as well as what that string means? @catenacyber

Reply:
> So, both of them are the same things, one a sticky buffer and one a value?

No

There is one integer which encodes a status as an enumeration, and a string which can be whatever...

See https://docs.suricata.io/en/latest/rules/integer-keywords.html#enumerations for the enumeration keywords and also `EnumStringU8` and such in the code

Reply:
ah ok. Thanks. I see you've already given feedback to change the string to something else. Thanks!